tests/unit/test_views.py

```diff
@@ -158,6 +160,21 @@ def test_new_analysis_request_post_not_logged_in(client, user):
     assert response.status_code == 302
 
 
+def test_analysis_request_output(client, user):
+    client.force_login(user)
+    analysis_request = AnalysisRequestFactory()
```

Comment:
Should this analysis request be linked to the `user` fixture?  I think you can do that with something like `AnalysisRequestFactory(user=user)`.